### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [rdoeffinger/Dictionary#136]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [QuickDic](https://play.google.com/store/apps/details?id=de.reimardoeffinger.quickdic&hl=it&gl=US)
3. Activate the registration from the Accessibility Scanner and use the QuickDic app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201119_095318](https://user-images.githubusercontent.com/25429624/99646348-a80b5f80-2a50-11eb-9a94-0e64e09ed6e0.png)

**Comment by rdoeffinger** [comment]
Increasing the contrast would make it look like regular text, which would just create confusion.
The text is not essential for being able to use the app either (for example there is the floating button with the search icon that will set the focus and it's possible to just type).
I can't see how making any change would do anything but make usability worse for anyone.
If someone has specific issues with using the app I am happy to discuss (which will usually involve discussing different approaches to improving it), but what a generic tool says is not helpful without careful review of its recommendations.
(I will note that Accessibility Scanner is not COMPLETELY useless: it has previously helped find items where a label was forgotten and misplaced, but nevertheless most of its suggestions are garbage)
